### PR TITLE
Updated flashcards on 26011 Long Test 1.md and made ETHICS.md

### PR DESCRIPTION
**Pull request by nigelhaim**
I have added new qustions on this Flashcards MarkDown file. Mostly on the widgets and some missed facts based form the PowerPoint of Mam Perol you can refer on my notes. 

https://github.com/nigelhaim/Notes_3rdYr/blob/main/Web%20and%20mobile%20programming.md

I have also created ETHICS flashcards, but I still have not finished it. I still haven't included (and skipped) List of Moral Intuitions (di ko alam kung paano siya gawin into flashcards) and stopped in Moral Reasoning. 

PS

Pre minsan tinitiiglan ko gumawa pero nagcocommmit ako time to time bago ako tumigil. Kung 12 am na pero di pa ako nakapag sabi na final commit consider mo nalang (or bago ka ma tulog TBH most likely bago ka ma tulog, Tulog na ako or inaantok na HAHAHAAHAH)

### CODE REVIEW

**Comment by 1123Alessandro** [comment]
AHAHSHSAHAAH GRABI ANG PAGOD MO PRE AH

**Review by 1123Alessandro — COMMENTED** [review]
WAO

**Review by 1123Alessandro — DISMISSED** [review]
oKS GOOD NA E2

File: Reviewers/26011 Long Test 1.md [hunk 1]
```diff
@@ -29,10 +29,40 @@ When you made any alteration in the code, the widget rebuilds its description by
 
 A widget is either stateful or stateless. If a widget can change when a user ==interacts with it==
 
+# Visible and Invisible Widgets
```
Comment (1123Alessandro) [inline Reviewers/26011 Long Test 1.md]: Tanggalin na to kasi magpapakita yung header sa flashcard, dead give away ng sagot

**Review by nigelhaim — COMMENTED** [review]
oks oks ngayon lang ako nakapag review HAHAHAH

**Review by 1123Alessandro — APPROVED** [review]

**Pull request merged**